css/style.css

```diff
@@ -16,4 +16,212 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ body {
+    margin: 0;
+  }
+  
+  body,
+  button {
+    color: var(--grey-dark);
+    font-size: 1rem;
+  }
+  
+  h1,h2,h3,h4,h5,h6 {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+  
+  p {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+  
+  .header,
+  .content,
+  .footer {
+    padding: 0 1.5rem;
+    margin: 0 auto;
+  }
+/* Header */
+ .header {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+  }
+  
+  .header__logo {
+    max-height: 40px;
+    width: 40px;
+  }
+  
+/* Navigation */
 
+.navigation__list {
+    display: flex;
+    list-style: none;
+    gap: 20px;
+  }
+  
+  .navigation__item {
+    padding: 0.5rem 0;
+  }
+  
+  .navigation__link {
+    color: var(--grey-dark);
+    font-weight: 300;
+    text-decoration: none;
+  }
+  
+  .navigation__link:hover {
+    color: #c05326;
+  }
+
+/* karma-intro */
+
+.karma-intro {
+    background-image: url("/HTML-CSS-Module-Project/img/first-background.jpg");
+    height: 420px;
+    color:white;
+    padding-top: 250px;
```

Comment:
Try to avoid using padding cause it could break on small screens.
Flexbox could help you out:
```
display: flex;
flex-direction: column;
justify-content: space-around;
```
space-around wil create three equal sections with spacing on top and bottom, and will be responsive